src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,164 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size = 0;
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            first = new Node<>(value, last, null);
+            size++;
+            return true;
+        }
+        if (size == 1) {
+            last = new Node<>(value, null, first);
+            first.next = last;
+            size++;
+            return true;
+        }
```

Comment:
try to think of the way of simplifying your code and completing this method without second if check